### DISCUSSION TITLE
stage

### DISCUSSION THREAD

**Discussion by TimOosterveer** [fdnd/docs.fdnd.nl#71] [Reflectie]
Stage verplicht maken van minimaal 10 weken

 - Op stage leer je veel op het gebied van frontend development wat je echt mist op school, technische kennis, (javascript, frameworks), samenwerking en omgaan met deadlines(inschattingen).

 - School zou dan meer opties moeten bieden voor stage bedrijven.

**Comment by ju5tu5** [comment]
Opgenomen in de tekst
